tests/collections/__init__.py

```diff
@@ -1,4 +1,4 @@
 """
 Tests cannot be written with a generic Collection. Thus we always need a specific one. See the base collection as an
 abstract class when being under test!
-"""
\ No newline at end of file
+"""
```

Comment:
Did the newline get added here? It doesn't appear in the diff

Reply:
GitHub shows a missing end of line with the red circle and the red bar. So this has indeed been added here.